platform/ios/scripts/bazel-generate-plists.sh

```diff
@@ -82,4 +82,5 @@ cat > platform/ios/bazel/config.bzl <<EOF
 APPLE_MOBILE_PROVISIONING_PROFILE_TEAM_ID = "$teamid"
 APPLE_MOBILE_PROVISIONING_PROFILE_NAME = "iOS Team Provisioning Profile: *"
 BUNDLE_ID_PREFIX = "$bundleidprefix"
+BUILD_MODE = "xcode"
```

Comment:
```suggestion
BUILD_MODE = "bazel"
```

Still having problems?

Reply:
Both work for me, I prefer "xcode" because of the information level during the build, that's all. 
It should not have been committed.